### PR TITLE
Add make target to rebuild CSS files

### PR DESCRIPTION
**Pull request by tsauerwein**
Run `make -f config/... less` to rebuild the CSS files.

### CODE REVIEW

**Comment by asaunier** [comment]
That's indeed useful, thanks!

File: Makefile [hunk 1]
```diff
@@ -29,7 +29,7 @@ help:
 	@echo "- clean			Remove generated files"
 	@echo "- cleanall		Remove all the build artefacts"
 	@echo "- compile-catalog	Compile the translation catalog"
-	@echo "- flake8			Run flake8 checker on the Python code"
+	@echo "- flake8		Run flake8 checker on the Python code"
```
Comment (gberaudo) [inline Makefile]: indentation

Comment (tsauerwein) [inline Makefile]: When I run `make help` it is perfectly aligned. Not for you?

Comment (gberaudo) [inline Makefile]: I see, the indentation appears unregular in this github page (and in vim), but you are right it is aligned when run in the shell.

**Pull request merged**